build.sh

```diff
@@ -11,6 +11,7 @@ usage()
     echo "clean - optional argument to force a clean build."
     echo "verbose - optional argument to enable verbose build output."
     echo "clangx.y - optional argument to build using clang version x.y."
+    echo "installcli - should download CLI and not use cached copy of the CLI."
```

Comment:
I do not think we need this option. 

The same sort of problem happens with nuget package cache once in a while, or with left-over build artifacts in binaries. `git clean` will fix all of these.